Lib/test/test_argparse.py

```diff
@@ -2193,9 +2193,9 @@ def test_wrong_argument_subparsers_no_destination_error(self):
         subparsers.add_parser('bar')
         with self.assertRaises(ArgumentParserError) as excinfo:
             parser.parse_args(('baz',))
-        self.assertRegex(
+        self.assertIn(
+            "error: argument {foo,bar}: invalid choice: 'baz', maybe you meant 'bar'? (choose from 'foo', 'bar')",
```

Comment:
I'd be interested in others' opinions around the verbiage here. It seems like we are using both "Maybe you meant" and "Did you mean" verbiage in the codebase. Not sure if we have any principle around this.